splunk/scripts/index_verification.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+logs_path="$SPLUNK_HOME/test_logs"
+endpoint="https://localhost:8089/services/data/inputs/oneshot"
+
+echo "Indexing logs..."
+for file in "$logs_path"/*; do
+  echo "Indexing $file"
+  curl -k -u admin:"$SPLUNK_PASSWORD" "$endpoint" -d name="$file"
+done
+
+response=$(curl -k -u admin:"$SPLUNK_PASSWORD" "$endpoint" -X GET -d output_mode=json)
+indexing_status=$(echo "$response" | jq -r '.paging.total')
```

Comment:
Can use `--exit-status` to do the boolean check from inside `jq` so you can do this whole thing in a single `if` statement instead of 3 different statements. Also no reason to store the `curl` response instead of piping in directly to `jq`, this is just allocating an unnecessary string.

Reply:
done